test/stages/stage1-resource-group.tf

```diff
@@ -1,6 +1,6 @@
 module "resource_group" {
   source = "github.com/cloud-native-toolkit/terraform-ibm-resource-group.git"
-
-  resource_group_name = var.resource_group_name
+  
+  resource_group_name    = module.resource_group.name
```

Comment:
@balasgit the resource_group module can't refer to its own output... The `resource_group_name` variable should be left as it was.

Reply:
I think this is the line you meant to update - https://github.com/terraform-ibm-modules/terraform-ibm-toolkit-vpe-gateway/blob/ef53b19a50531451ec6b04b17dfdd93867114b69/test/stages/stage1-icr.tf#L4